sbomtool/internal/commands/merge/validator.go

```diff
@@ -0,0 +1,82 @@
+// Package merge provides validation functionality for merge operations.
```

Comment:
In other commits, the validations are performed by individual functions rather than by struct. For consistency, you should choose one approach (either an interface with implementations, or just plain functions)